recitation_3/memorypointers.md

```diff
@@ -0,0 +1,282 @@
+
+# Memory and Pointers #
+
+Understanding pointers and the ins and outs of memory usage in your program is a fundamental part of what makes you a strong C programmer! This note breaks down important elements that go into this understanding.   
+
+
+## Memory ##
+
+### Address Space ###
+
+Every process, i.e a running program, gets 512G of virtual memory space. The memory layout is given in the diagram below.
```

Comment:
this depends on a lot of things, if you want to keep the specific number you should say "on the AP Server"